lib/plugins/aws/deploy/lib/ensure-valid-bucket-exists.js

```diff
@@ -31,7 +31,7 @@ module.exports = {
         });
       } catch (err) {
         throw new ServerlessError(
-          `Could not locate deployment bucket. Error: ${err.message}`,
+          `Could not locate deployment bucket: (${this.bucketName}). Error: ${err.message}`,
```

Comment:
ah.. sorry, I've written _parenthesis_, while meant quotes 🤦  Can you update it, so it's wrapped in `"` (double quotes) ?

Reply:
lol.  🤣   all good; patch updated.